Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM python:3 as base
```

Comment:
Python versions aren't always backwards compatible. So to avoid being surprised by an update to 3.10, it would be better to specify `python:3.9`

Reply:
Thanks will do in Module 7